ios/Flutter/flutter_export_environment.sh

```diff
@@ -1,14 +1,13 @@
 #!/bin/sh
```

Comment:
this too i guess because we'll both be changing it

Reply:
This one seems to be there under iOS/Xcode related, why isn't it working? Is it because of the additional ** at the beginning of the path?

Reply:
I'm not sure.. Where do you see the **? 😕

Reply:
Line 61 of the .gitignore

Reply:
Oh yeah. I'm not sure, gitignore never seems to work properly!
Never mind let's merge anyway. Thanks!